src/policy/fees.cpp

```diff
@@ -548,14 +548,14 @@ CBlockPolicyEstimator::CBlockPolicyEstimator(const fs::path& estimation_filepath
 
     AutoFile est_file{fsbridge::fopen(m_estimation_filepath, "rb")};
 
-    // Whenever the fee estimation file is not present return early
     if (est_file.IsNull()) {
         LogPrintf("%s is not found. Continue anyway.\n", fs::PathToString(m_estimation_filepath));
         return;
     }
 
     std::chrono::hours file_age = GetFeeEstimatorFileAge();
-    // fee estimate file must not be too old to avoid wrong fee estimates.
+    // Serving stale fee estimates can cause transactions to become stuck in the bottom of the mempool in the worst case.
+    // Don't do that.
```

Comment:
I think I know what you mean, but imo transactions being stuck is a little bit of a jump from feerate underestimation. Maybe "Serving stale fee estimates may lead to overpaying in fees or not meeting minimum feerate requirements in the worst case." Or just delete this comment, as the code is self-explanatory and there is already a comment above `MAX_FILE_AGE`.